administrators.rst

```diff
@@ -759,6 +759,48 @@ Fields:
    the slash. Do so with caution and be prepared to put the leading-slash
    back if there is any impact on storing AIPs as a result.
 
+Debugging S3
+""""""""""""
+
+There are times when the S3 storage adapter needs to be debugged. For example,
+if a transfer isn't able to complete because it cannot reach your S3
+implementation.
+
+The S3 adapter written for the Storage Service relies heavily on the `Boto3`_
+S3 SDK (Software Development Kit). The library is hosted on GitHub and the
+GitHub issues for Boto3 is a good place to start when trying to understand
+potential upstream issues.
+
+In the Storage Service, debug logging can be increased for the Boto3 adapter.
+There is more information about overriding the Storage Service defaults in the
+`installation README.md`_.
+
+In the logging configuration, administrators should be able to find entries for
```

Comment:
Maybe worth linking to the logging configuration file in the source code?

Reply:
Nice idea, will do!